add note for All groups option (#13929)
DataDog/documentation: content/en/monitors/notify/downtimes.md

If a multi-alert monitor is included, it is only silenced for groups covered by the scope. For example, if a downtime is scoped for `host:X` and a multi-alert is triggered on both `host:X` and `host:Y`, Datadog generates a monitor notification for `host:Y`, but not `host:X`.

> If you would like to include all groups in the scope of a Downtime which applies to multi-alert monitors, you should select `All` for the `Group scope`.

Reviewer: ```suggestion
To include all groups in the scope of a Downtime that applies to multi-alert monitors, select `All` for the `Group scope`.
```
